portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,95 @@
-#content {
+.content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  margin-bottom: 20px;
+  width: 75%;
 }
 
-#greeting-container {
+.projpic {
+  width: 30%;
+  padding-left: 10px;
+  padding-right: 10px;
   margin-top: 20px;
+  margin-bottom: 20px;
+}
+.randomFact {
```

Comment:
It's good to adopt a consistent formatting. Elsewhere you are leaving one blank line between style definitions. I think you should have a blank here too.

Reply:
Yes, I totally agree with that. Thanks for catching it.

Reply:
You forgot to fix? Or missed including the commit in this PR.